docs/config.md

```diff
@@ -30,6 +30,9 @@ OPTIONS
   -u, --user=user                   [default: current] User used to run docker images when creating configuration files
                                     like certificates or nemesis block. "current" means the current user.
 
+  --backupSync                      It downloads a backup with the Mongo and RocksDb databases for faster
```

Comment:
It would be nice to say WHERE does the download come from. Symbol servers? Can it be configured?

Reply:
Description updated. The location is in a preset configuration. Users could private a custom local or remote location.

Reply:
The where depends on the network. Atm we only have one backup, the testnet one. Public net would have a different backup.